packages/admin/cms-admin/src/generator/future/generateForm/generateFormField.ts

```diff
@@ -209,9 +209,8 @@ export function generateFormField({
             }
         });
 
-        if (config.inputType === "radio") {
-            code = `
-            <RadioGroupField
+        if ((required && values.length <= 5) || config.inputType === "radio") {
```

Comment:
If I explicitly set `inputType: "select"` but it's not required or has less than 5 items, it will still render the radio-field. Also using `else if` instead of just `else` for exactly the opposite isn't necessary and could theoretically lead to nothing being rendered at all. 

Maybe the following logic makes more sense:

```ts
const renderAsRadio = config.inputType === "radio" || (required && values.length <= 5 && config.inputType !== "select");

if (renderAsRadio) {
    // render RadioGroupField
} else {
    // render SelectField
}
```

Reply:
Yeah, I see my logic error now. Thanks for pointing that out!